wasp-ai/src/client/pages/UserPage.jsx

```diff
@@ -0,0 +1,101 @@
+import { useQuery } from "@wasp/queries";
+import getProjectsByUser from "@wasp/queries/getProjectsByUser";
+import { Link } from "@wasp/router";
+import { Color } from "../components/Color";
+import { Header } from "../components/Header";
+import { PiDownloadDuotone } from "react-icons/pi";
+import logout from "@wasp/auth/logout";
+import { FiLogOut } from "react-icons/fi";
+import { format } from "timeago.js";
+import { StatusPill } from "../components/StatusPill";
+import { getColorValue, getStatusName, getStatusText } from "./StatsPage";
+
+export function UserPage({ user }) {
+  const { data: projects, isLoading, error } = useQuery(getProjectsByUser);
+
+  if (isLoading) "Loading Projects...";
```

Comment:
Hm hoe does this work? Does this work? I would expect these two lines to do nothing at all actually, they have string, which is an expression, playing the role of the statement, which means nothing happens.

It is like writing
```js
console.log('Hi');
'hellou'
```

Basicaly this second line gets evaluated and then value is just dropped.

Reply:
I don't think it is, that is not how JS works.

These lines are basically no-op -> they don't do anything, you could remove them from you program and its functioning won't change.

Try this:
```js
function vince (x) {
    if (x === 1) "one";
    if (x === 2) "two";
    return "not one or two";
}
```

Whichever argument you give it, it will return `"not one or two"`.

JS program (and same is for most other languages, like C++, python, Java, ...) is a list of statements, at the top level. So a bunch of statements:

STMT
STMT
STMT

Statement can be something like `if () {}` or `for ...` or `import ...` or `function foo () {...}` and so on.
Then we also have expressions, which are stuff like `1` or `1 + 2` or `vince(3)` and so on.

Statements usually have expressions in them as their parts.

But, just a standalone expression also counts as statement!

That makes sense, because that allows you to call a function alone in its line, because function call is an expression.
But it also allows stuff like:

```js
console.log("hi");

"hi";

return 3;
```

What is that "hi" doing there on its own? Well, nothing really. It is a statement that is really just an expressions (because an expression is also a valid statement), and interpreter / compiler will go line by line, evaluate console.log("hi") and write "hi" to stdout, then it will evaluate "hi" and say "ok, this is 'hi', but nothing to do with it so I will just leave it here", and then `return 3` ends the function and so on.

```
if (...) "hi";
```

will have the same effect as just
```
"hi"
```

which is nothing, it will just execute only sometimes.

If somehow possibly I got this wrong, I apologize, but I am 99.99% confident in what I am writing here. I am leaving 0.01% because you never know.

Reply:
whoop! i forgot to add `return`

so it should be 
```
if (isLoading) return "Loading..."
```

Reply:
Damn :D! Yes that is fine :D.